dotCMS/src/integration-test/java/com/dotmarketing/portlets/htmlpages/business/render/HTMLPageAssetRenderedAPIImplIntegrationTest.java

```diff
@@ -1949,7 +1954,11 @@ public void injectJSCodeWithoutHead() throws WebAssetException, DotDataException
     /**
      * Method to test: {@link HTMLPageAssetRenderedAPI#getPageHtml(PageContext, HttpServletRequest, HttpServletResponse)}
      * When: Try to render a page with a specific {@link Variant}} and a specific {@link Language}
+<<<<<<< HEAD
+     * and the page had a Contentlet that has a version in the variant but not in the DEFAULT Variant
+=======
```

Comment:
fix this conflicts

Reply:
done https://github.com/dotCMS/core/pull/24209/commits/7135f6d5cf901210a453eecb927ce359aa8d7dc2#diff-aac9fd429a1e8f9ab8efad97d4259f8be79b41078a9c0750704c907760f2d85cL1957